src/server/api/controllers/topics.controller.js

```diff
@@ -26,7 +27,54 @@ const getTopicById = async (id) => {
   }
 };
 
+const editTopic = async (topicId, body) => {
+  const timestamp = moment().format('YYYY-MM-DD HH:mm:ss');
+  return knex('topics').where({ id: topicId }).update({
+    topic_name: body.topicName,
+    created_at: timestamp,
```

Comment:
When editing, `created_at` should not change. Because that would mean you would forget the time of creation.